src/proposals/proposal.cpp

```diff
@@ -168,9 +170,11 @@ namespace hypha
         asset votes_fail = tally.getDocument().getContentWrapper().getOrFail(common::BALLOT_DEFAULT_OPTION_FAIL.to_string(), VOTE_POWER)->getAs<eosio::asset>();
 
         asset total = votes_pass + votes_abstain + votes_fail;
+        // pass / ( pass + fail ) > alignmentFactor
+        bool passesAlignment = votes_pass > (alignmentFactor * (votes_pass + votes_fail));
         bool passed = false;
```

Comment:
What's the use for this variable?

Reply:
ah - it's no longer used it seems, thanks!

```suggestion
```